TAF/testData/core-metadata/resource-profile.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This is not a valid json file which contains duplicate `resource` properties. Please refer to the json files in the `testData`